src/main/java/org/jabref/gui/fieldeditors/KeywordsEditorViewModel.java

```diff
@@ -0,0 +1,52 @@
+package org.jabref.gui.fieldeditors;
+
+import javax.swing.undo.UndoManager;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+import javafx.util.StringConverter;
+
+import org.jabref.gui.autocompleter.SuggestionProvider;
+import org.jabref.logic.integrity.FieldCheckers;
+import org.jabref.model.entry.Keyword;
+import org.jabref.model.entry.field.Field;
+import org.jabref.preferences.PreferencesService;
+
+public class KeywordsEditorViewModel extends AbstractEditorViewModel {
+
+    private final ListProperty<Keyword> keywordListProperty;
+
+    public KeywordsEditorViewModel(Field field,
+                                   SuggestionProvider<?> suggestionProvider,
+                                   FieldCheckers fieldCheckers,
+                                   PreferencesService preferencesService,
+                                   UndoManager undoManager) {
+
+        super(field, suggestionProvider, fieldCheckers, undoManager);
+
+        Character keywordSeparator = preferencesService.getBibEntryPreferences().getKeywordSeparator();
+        keywordListProperty = new SimpleListProperty<>(FXCollections.observableArrayList());
+    }
+
+    public ListProperty<Keyword> keywordListProperty() {
+        return keywordListProperty;
+    }
+
+    public StringConverter<Keyword> getStringConverter() {
+        return new StringConverter<>() {
+            @Override
+            public String toString(Keyword keyword) {
+                if (keyword != null) {
+                    return keyword.get();
+                }
+                return "";
```

Comment:
When can be a keyword be null? - I think, because below in the code `null`  is passed. In modern Java, null should not be used.

If it really needs to be kept:

In JabRef, we have the pattern "fail fast". Therefore, reverse the if

```suggestion
                if (keyword == null) {
                    return "";
                }
                return keyword.get();
```

Reply:
> When can a keyword be null?

I'm not sure when a keyword can be `null`, but I think it will never be `null`. I based this on the following examples:

- https://github.com/dlsc-software-consulting-gmbh/GemsFX/blob/8e353ee0dfa370d37d8c43bb244070590390560a/gemsfx-demo/src/main/java/com/dlsc/gemsfx/demo/TagsFieldApp.java#L159-L166
- https://github.com/JabRef/jabref/blob/94928dd8e0811e962be8c7ccf7145835dbe969a7/src/main/java/org/jabref/gui/fieldeditors/LinkedEntriesEditorViewModel.java#L42-L47